### PR TITLE
10 clean up

### PR DESCRIPTION
**Pull request by ManuelJF**
delete unused code to resolve warnings from the console of the browser
and delete all unless console logs 👍

### CODE REVIEW

File: src/registerServiceWorker.js [hunk 1]
```diff
@@ -65,20 +65,16 @@ function registerValidSW(swUrl) {
               // the fresh content will have been added to the cache.
               // It's the perfect time to display a "New content is
               // available; please refresh." message in your web app.
-              console.log('New content is available; please refresh.');
```
[the remark this answers is not in the document]
Comment (EricTurf) [reply src/registerServiceWorker.js]: Agreed, please undo these changes and I can merge this PR :)

@ra100 Still interested in adding es-lint?

[the remark this answers is not in the document]
Comment (ManuelJF) [reply src/registerServiceWorker.js]: I think putting a message under the board as soon as I need to refresh would be perfect

[the remark this answers is not in the document]
Comment (ra100) [reply src/registerServiceWorker.js]: @EricTurf I'll add eslint, when this is merged, so we don't have to solve merge conflicts

[the remark this answers is not in the document]
Comment (ManuelJF) [reply src/registerServiceWorker.js]: @EricTurf @ra100 done!

**Pull request merged**